modules/nf-core/annotsv/annotsv/main.nf

```diff
@@ -39,11 +39,15 @@ process ANNOTSV_ANNOTSV {
         ${small_variants} \\
         ${fp_snv} \\
         ${transcripts} \\
-        -outputFile ${prefix}.tsv \\
+        -outputFile ${prefix}.raw.tsv \\
         -SVinputFile ${sv_vcf} \\
         ${args}
 
     mv *_AnnotSV/* .
+    awk 'BEGIN { FS=OFS="\t" } { if (NR > 1 && NF >= 8) \$1 = \$1 "_" NR; print }' ${prefix}.raw.tsv > ${prefix}.tsv
+
+    variantconvert convert -i ${prefix}.tsv -o ${prefix}.vcf -fi annotsv -fo vcf -c /usr/local/share/python3/variantconvert/configs/GRCh38/annotsv3_from_vcf.json
```

Comment:
nice tool, didn't know that one

Reply:
It's the tool AnnotSV uses natively to convert their output TSV to VCF format, but I have to do it this way due to the ID issue in AnnotSV